content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -43,7 +43,12 @@ The records grid is quite different than the products' one. Thereafter are the s
 - Finally, the grid can display up to 500 results using an **infinite scroll**. If you have more results, please filter your search. When reaching this limit, the grid displays the message *"You just scrolled through the first 500 results out of 1450 records. To see more, try with more specific search criteria."*
 
 ::: info
-In the records grid, you will see only certain types of attributes: code, text and image.
+In the records grid in 3.0, you will see only certain types of attributes: code, text and image.  
+_Since 3.1_, the values of the attributes types single/multiple options and reference entity single/multiple links are also displayed in the records grid.
+:::
+
+::: info
+The order of the columns in the records grid is "Image, Label, Code, Complete", the other attributes are then ordered according to their order of creation in the "Attributes" tab.
```

Comment:
The columns' order in the records grid is frozen, it cannot be changed. By default, the grid's column order is `Image`, `Label`, `Code`, `Complete`. If you add other attributes, under the `Attributes` tab, the grid will display them according to their order of creation.